src/openrct2/ride/Vehicle.cpp

```diff
@@ -10052,10 +10052,10 @@ void vehicle_claxon(const rct_vehicle * vehicle)
     rct_ride_entry* rideEntry = get_ride_entry(vehicle->ride_subtype);
     switch (rideEntry->vehicles[vehicle->vehicle_type].sound_range)
     {
-    case 3:
+    case SOUND_TRAIN_WHISTLE:
```

Comment:
This isn't going to work — these constants represent the sound files themselves. Their values are much higher than 3 and 4.

Reply:
I looked in vehicle.h to try and find the numbers when I first started this PR. I didn't find them there so I thought they didn't exist.

Reply:
Indeed, they probably don't exist yet.

Reply:
You need to create new constants.

```
enum sound_range
{
    SOUND_RANGE_TRAIN = 3,
    SOUND_RANGE_TRAM = 4,
}
```